wdl/Vapor.wdl

```diff
@@ -20,15 +20,18 @@ workflow Vapor {
     String sv_base_mini_docker
     String sv_pipeline_docker
 
-    RuntimeAttr? runtime_attr_vapor 
+    RuntimeAttr? runtime_attr_vapor
     RuntimeAttr? runtime_attr_bcf2vcf
     RuntimeAttr? runtime_attr_vcf2bed
     RuntimeAttr? runtime_attr_split_vcf
     RuntimeAttr? runtime_attr_concat_beds
     RuntimeAttr? runtime_attr_LocalizeCram
+
+    Boolean save_plots  # New input to control whether plots are final output
+    File? NONE_FILE_ # Create an empty file
```

Comment:
```suggestion
    File? NONE_FILE_  # Create a null file - do not use this input
```

Reply:
Thank you for that clarification, the edit was made.